CONTRIBUTING.md

```diff
@@ -43,4 +88,26 @@ For a deeper discussion, see: https://github.com/open-telemetry/opentelemetry-sp
 ## Styleguide
 
 * docstrings should adhere to the Google styleguide as specified
-  with the [napolean extension](http://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#google-vs-numpy) extension in [Sphinx](http://www.sphinx-doc.org/en/master/index.html).
\ No newline at end of file
+  with the [napolean extension](http://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#google-vs-numpy) extension in [Sphinx](http://www.sphinx-doc.org/en/master/index.html).
+
+## Become a Collaborator
```

Comment:
are these instructions more general to OTel? would it make more sense to reference a more general doc?

Reply:
Good point, I was thinking about this, too.
I decided to go with Python first, and then use the published doc as a proposal to see if other languages are willing to follow. If yes, would definitely move it to a common place.

Reply:
This all LGTM, but I agree that we need a consistent policy across the org.